php/Terminus/KLogger.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Terminus;
+
+use Terminus\Runner;
+use Katzgrau\KLogger\Logger;
+use Psr\Log\LogLevel;
+
+//TODO: Change this classname to Logger once the old logger is fully replaced
+class KLogger extends Logger {
+
+  /**
+   * Class constructor. Feeds in output destination from env vars
+   *
+   * @param [array]  $options           Options for operation of logger
+   *        [array] config Configuration options from Runner
+   * @param [string] $logDirectory      File path to the logging directory
+   * @param [string] $logLevelThreshold The LogLevel Threshold
+   */
+  public function __construct(
+    array $options = array(),
+    $logDirectory = 'php://stdout',
+    $logLevelThreshold = LogLevel::INFO
+  ) {
+    $config = $options['config'];
+    unset($options['config']);
+
+    if (isset($_SERVER['TERMINUS_LOG_DIR'])) {
+      $logDirectory = $_SERVER['TERMINUS_LOG_DIR'];
+    } elseif ($config['silent']) {
+      $logDirectory = '/var/log/terminus/';
```

Comment:
I don't imagine log files are in the same place on Windows, so this probably has to change.